modules/arm_plugin/Dockerfile.RPi64_focal

```diff
@@ -0,0 +1,108 @@
+    echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ focal multiverse >> /etc/apt/sources.list && \
+    echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ focal-updates multiverse >> /etc/apt/sources.list && \
+    echo deb [arch=amd64] http://archive.ubuntu.com/ubuntu/ focal-backports main restricted universe multiverse >> /etc/apt/sources.list && \
+    echo deb [arch=amd64] http://security.ubuntu.com/ubuntu/ focal-security main restricted >> /etc/apt/sources.list && \
+    echo deb [arch=amd64] http://security.ubuntu.com/ubuntu/ focal-security universe >> /etc/apt/sources.list && \
+    echo deb [arch=amd64] http://security.ubuntu.com/ubuntu/ focal-security multiverse >> /etc/apt/sources.list && \
+    echo deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports/ focal main >> /etc/apt/sources.list && \
+    echo deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports/ focal universe >> /etc/apt/sources.list && \
+    echo deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports/ focal-updates main >> /etc/apt/sources.list && \
+    echo deb [arch=arm64] http://ports.ubuntu.com/ubuntu-ports/ focal-security main >> /etc/apt/sources.list && \
+    dpkg --add-architecture arm64 && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends \
+        locales \
+        apt-transport-https \
+        debian-archive-keyring \
+        software-properties-common \
+        gnupg \
+        p7zip-full \
+        chrpath \
+        curl \
+        wget \
+        git \
+        scons \
+        cmake \
+        build-essential \
+        crossbuild-essential-arm64 \
+        python3-dev \
+        python3-numpy \
+        python3-pip \
+        libpython3-dev:arm64 \
+        libprotoc-dev \
+        protobuf-compiler \
+        libffi-dev \
+        libssl-dev \
+        libusb-1.0-0-dev:arm64 \
+        libgtk-3-dev:arm64 \
+        libavcodec-dev:arm64 \
+        libavformat-dev:arm64 \
+        libswscale-dev:arm64 \
+    && \
+    locale-gen en_US.UTF-8 && \
+    pip3 install cython && \
+    rm -rf /var/lib/apt/lists/*
+
+# To cross-compile Python3.7 we need to first compile it for the host
+RUN curl -O https://www.python.org/ftp/python/3.7.9/Python-3.7.9.tar.xz && \
+    tar -xf Python-3.7.9.tar.xz && \
+    cd Python-3.7.9 && ./configure && make -j4 && make altinstall && \
+    curl https://bootstrap.pypa.io/get-pip.py | python3.7 - --no-cache-dir pip==20.2.2 numpy==1.19.5 cython
+
+# Cross-compile Python3.7 for ARM
+RUN cd Python-3.7.9 && make distclean && \
+    ./configure \
+        --host=aarch64-linux-gnu \
+        --build=x86_64-linux-gnu \
+        --disable-ipv6 \
+        --enable-shared \
+        --prefix=/opt/python3.7_arm \
+        ac_cv_file__dev_ptmx=no \
+        ac_cv_file__dev_ptc=no && \
+    make -j4 && make altinstall
+
+RUN wget https://www.cmake.org/files/v3.13/cmake-3.13.4.tar.gz && \
```

Comment:
Ubuntu Focal has cmake 3.16.3 so building from source is unnecessary

Reply:
removed cmake compilation